src/content/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1570.mdx

```diff
@@ -0,0 +1,13 @@
+---
+subject: Infrastructure agent
+releaseDate: '2024-09-02'
+version: 1.57.0
+---
+
+We've released a new version of the agent. Follow standard procedures to [update the Infrastructure agent](https://docs.newrelic.com/docs/infrastructure/install-configure-manage-infrastructure/update-or-uninstall/update-infrastructure-agent).
+New Relic recommends that you upgrade the agent regularly, every 3 months at a minimum. As of this release, the oldest supported version is [Infrastructure agent 1.47.0](https://docs.newrelic.com/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes/new-relic-infrastructure-agent-1470/).
+
+## Changed
+
+* New health status check and report for the status api [#1908](https://github.com/newrelic/infrastructure-agent/pull/1908)
+* Exclude matching metrics [#1914](https://github.com/newrelic/infrastructure-agent/pull/1914)
```

Comment:
```suggestion
* New local health status (/v1/status/health) checks for valid credentials to connect to NR backend [#1908](https://github.com/newrelic/infrastructure-agent/pull/1908)
* Ability to define exclusion lists for process monitoring using the `exclude_matching_metrics` [setting] (https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings/#exclude-matching-metrics) [#1914](https://github.com/newrelic/infrastructure-agent/pull/1914)
```

Reply:
I edited the github release notes soon after you created this PR, if possible, let's sync them.